src/main/java/graphql/schema/GraphQLDirective.java

```diff
@@ -201,11 +218,20 @@ public Builder clearValidLocations() {
         }
 
         public Builder argument(GraphQLArgument argument) {
-            Assert.assertNotNull(argument, "argument must not be null");
+            assertNotNull(argument, "argument must not be null");
             arguments.put(argument.getName(), argument);
             return this;
         }
 
+        public Builder arguments(List<GraphQLArgument> arguments) {
```

Comment:
consistency - you called it replaceDirectives above.  Should it be replaceXXX everywhere?

Reply:
done